fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -162,7 +163,11 @@
 
     MISSING_JOB_ID("Bard experienced an internal error. Sorry.", "Missing id for job row %s"),
 
-    INVALID_ASYNC_AFTER( "Invalid 'asyncAfter' parameter: '%s'. 'asyncAfter' must be either 'never' or an integer number of milliseconds.");
+    INVALID_ASYNC_AFTER( "Invalid 'asyncAfter' parameter: '%s'. 'asyncAfter' must be either 'never' or an integer number of milliseconds."),
+
+    FILTER_JOBFIELD_UNDEFINED("Filter JobField '%s' does not exist."),
+
+    JOBFIELD_NOT_PRESENT_IN_JOB_META_DATA("JobField '%s' is not a part of job meta data. The possible fields to filter on are '%s'");
```

Comment:
I think treating the two as the same is fine, especially since what "valid" means isn't particularly well defined. We have a default set of fields, but the `JobRow` itself has no notion of valid or invalid fields.